research/picai/fl_nnunet/run_fl_single_node.slrm

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+#SBATCH --gres=gpu:a40:1
+#SBATCH --qos=normal
+#SBATCH --mem=64GB
+#SBATCH -c 16
+#SBATCH --time=16:00:00
+#SBATCH --signal=B:USR1@60
+#SBATCH --job-name=fedavg_picai
+#SBATCH --output=fedavg_picai_%j_%x.out
+#SBATCH --error=fedavg_picai_%j_%x.err
+
+###############################################
+# Usage:
+#
+# Notes:
+# 1) The sbatch command above should be run from the top level directory of the repository.
+# 2) The logging directories need to ALREADY EXIST. The script does not create them.
+###############################################
+
+# Set NCCL options
+# export NCCL_DEBUG=INFO
+# NCCL backend to communicate between GPU workers is not provided in vector's cluster.
+# Disable this option in slurm.
+export NCCL_IB_DISABLE=1
+
+if [[ "${SLURM_JOB_PARTITION}" == "t4v2" ]] || \
+    [[ "${SLURM_JOB_PARTITION}" == "rtx6000" ]]; then
+    echo export NCCL_SOCKET_IFNAME=bond0 on "${SLURM_JOB_PARTITION}"
+    export NCCL_SOCKET_IFNAME=bond0
+fi
+
+# Process Inputs
+
+CONFIG_PATH=$1
```

Comment:
Perhaps add some documentation on these inputs and maybe an example command to launch the slurm similar to what we have in the flamby scripts?